.github/workflows/arc-mlir.yml

```diff
@@ -35,10 +35,9 @@ jobs:
       run: |
         cd arc-mlir
         ./arc-mlir-build
-        for COMMIT in $COMMITS; do
+        for COMMIT in $(git rev-list mlir..HEAD); do
```

Comment:
Add something like  `echo Testing $COMMIT` here as otherwise we have no clue which commit failed

Reply:
I checked the container, it seems like there is just one commit: 🤔 
```
root@48f46d5ba2ba:~/Workspace/github-actions-runner/_work/arc/arc# git log
commit 8d33340a6ab4dd4019e92a60cba30bdbe7422013 (grafted, HEAD, pull/125/merge)
Author: Frej Drejhammar <frej.drejhammar@gmail.com>
Date:   Wed Jul 29 13:32:40 2020 +0000

    Merge 4b91be891ccd97b54c994af2f62c893ce2f4e6ff into e5ffeb5b802c5535a55aa57c4d898427affaf2b6
root@48f46d5ba2ba:~/Workspace/github-actions-runner/_work/arc/arc# git status
HEAD detached at pull/125/merge
nothing to commit, working tree clean
root@48f46d5ba2ba:~/Workspace/github-actions-runner/_work/arc/arc# git branch
* (HEAD detached at pull/125/merge)
```

Reply:
Didn't you clone with depth=0?

Reply:
> I asked on the GitHub actions board and they just sent a response

I only get less and less impressed by github, the more I use it. They clearly don't support a git patch-series in any meaningful way.

Reply:
I thought 0 = fetch everything
https://github.com/actions/checkout
```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

Reply:
You're right, I was off-by-one.